proto/cel/policy/BUILD.bazel

```diff
@@ -0,0 +1,75 @@
+load("@com_google_protobuf//bazel:proto_library.bzl", "proto_library")
+load("@com_google_protobuf//bazel:java_proto_library.bzl", "java_proto_library")
+load("@com_google_protobuf//bazel:java_lite_proto_library.bzl", "java_lite_proto_library")
+load("@io_bazel_rules_go//proto:def.bzl", "go_proto_library")
+load("@com_google_protobuf//bazel:cc_proto_library.bzl", "cc_proto_library")
+load("@com_google_protobuf//bazel:upb_c_proto_library.bzl", "upb_c_proto_library")
+load("@com_google_protobuf//bazel:py_proto_library.bzl", "py_proto_library")
+
+package(default_visibility = ["//visibility:public"])
+
+##############################################################################
+# Common
+##############################################################################
+
+proto_library(
+    name = "policy_proto",
+    srcs = [
+        "policy.proto",
+    ],
+    strip_import_prefix = "/proto",
+)
+
+##############################################################################
+# Java
+##############################################################################
+
+java_proto_library(
+    name = "policy_java_proto",
+    deps = [":policy_proto"],
+)
+
+java_lite_proto_library(
+    name = "policy_java_proto_lite",
+    deps = [":policy_proto"],
+)
+
+###############################################################################
+## Go
+###############################################################################
+
+go_proto_library(
+    name = "policy_go_proto",
+    importpath = "cel.dev/expr",
```

Comment:
Please update the import path to be `cel.dev/policy`

Reply:
Done.